reference-architecture/osp-cli/OSv3.yml.template

```diff
@@ -0,0 +1,36 @@
+deployment_type: openshift-enterprise
```

Comment:
I think at this point we want to avoid breaking existing users/docs, so I have not heard any requests to rename from OSEv3 (yet).